css/index.css

```diff
@@ -1,3 +1,489 @@
+    width: 75%;
+    margin: 0 auto;
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+}
+
+nav a {
+    text-decoration: none;
+    color: lightgray;
+    padding-right: 60px;
+}
+
+.demand-main {
+    display: flex;
+    justify-content: space-between;
+}
+
+.demand {
+    display: flex;
+    flex-direction: column;
+    font-weight: bold;
+    text-align: center;
+    align-items: center;
+    margin-bottom: 50px;
+}
+
+h1 {
+    font-size: 90px;
+    display: block;
+}
+
+.demand button {
+    display: block;
+    font-size: 20px;
+    background-color: white;
+    padding: 5px;
+    border-radius: 5%;
+    width: 150px;
+
+
+}
+
+button {
+    border: 1px solid black;
+}
+
+.features-about {
+    display: flex;
+    justify-content: space-around;
+    margin: 30px 0px;
+    line-height: 20px;
+}
+
+.services-vision {
+    display: flex;
+    justify-content: space-around;
+    margin: 30px 0px;
+    line-height: 20px;
+}
+
+.services {
+    margin: 0px 20px 0px 0px;
+}
+
+.product {
+    margin: 0px 20px 0px 0px;
+}
+
+.features {
+    display: flex;
+    flex-direction: column;
+    margin-right: 20px;
+
+}
+
+.about {
+    display: flex;
+    flex-direction: column;
+
+}
+
+.main-header header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.main-header {
+    padding: 30px 0;
+
+}
+
+.section {
+    display: flex;
+    justify-content: space-between;
+}
+
+.sectiion2 {
+    display: flex;
+}
+
+.footer {
+    text-align: center;
+    margin: 20px;
+}
+
+.container1 {
+    display: flex;
+}
+
+.container2 {
+    display: flex;
+}
+
+.container3 {
+    display: flex;
+}
+
+.container4 {
+    display: flex;
+
+}
+
+.box {
+    border: 1px solid black;
+    margin: 10px;
+    background-color: lightgray;
+    display: block;
+    width: 50%;
+    padding: 10px
+}
+
+.facts {
+    margin: 0px 40px 0px 0px;
+}
+
+ul {
+    margin: 20px 0px;
+}
+
+ul li {
+    margin: 20px 20px;
+    list-style: disc;
+}
+
+.middle-img {
+    width: 100%;
+}
+
+@media(max-width: 800px) {
+    header {
+        flex-direction: column;
+    }
+
+    .header-img {
+        display: none;
+    }
+
+    .logo {
+        order: 1;
+    }
+
+    nav {
+        order: 2;
+    }
+
+    .demand-main {
+        justify-content: center;
+    }
+
+    header {
+        display: flex;
+        justify-content: center;
+    }
+}
+
+@media(max-width: 500px) {
+    header {
+        flex-direction: column;
+    }
+
+    .header-img {
+        display: none;
+    }
+
+    .logo {
+        order: 1;
+    }
+
+    nav {
+        order: 2;
+    }
+
+    .demand-main {
+        justify-content: center;
+    }
+
+    header {
+        display: flex;
+        justify-content: center;
+    }
+
+    nav {
+        flex-direction: column;
+    }
+
+    nav a {
+        padding: 10px;
+        border-bottom: 1px solid lightgray;
+        border-top: 1px solid lightgray;
+        width: 500px;
+        padding-left: 225px;
```

Comment:
This puts all the text near the center, but all the test is still justified left. Remember, `text-align`?